File: python/readable_time.py
def make_readable(seconds):
	s = seconds % 60
	m = (seconds / 60) % 60
	h = (seconds / 3600)

	time = str(h) if (h>=10) else "0"+str(h)
	time += ":" + (str(m) if (m>=10) else "0"+str(m))
	time += ":" + (str(s) if (s>=10) else "0"+str(s))
	return time

print(make_readable(0) + "   00:00:00")
print(make_readable(5) + "   00:00:05")
print(make_readable(60) + "   00:01:00")
print(make_readable(86399) + "   23:59:59")
print(make_readable(359999) + "   99:59:59")

File: nodejs/readable_time.js
function humanReadable(seconds)
{
	var s = seconds % 60;
	var m = parseInt(seconds / 60) % 60;
	var h = parseInt(seconds / 3600);

	var time = (h>=10) ? ""+h : "0"+h;
	time += ":" + ((m>=10) ? ""+m : "0"+m);
	time += ":" + ((s>=10) ? ""+s : "0"+s);
	return time;
}

console.log(humanReadable(0) + "   00:00:00")
console.log(humanReadable(5) + "   00:00:05")
console.log(humanReadable(60) + "   00:01:00")
console.log(humanReadable(86399) + "   23:59:59")
console.log(humanReadable(359999) + "   99:59:59")
